server/api/cart.js

```diff
@@ -61,18 +62,40 @@ router.post('/cart/:id', requireToken, async (req, res, next) => {
   }
 });
 
-router.delete('/cart/:id', requireToken, async (req, res, next) => {
+router.put('/cart', requireToken, async (req, res, next) => {
   try {
     if (!req.user) {
       throw new Error('Unauthorized');
     }
-    const cart = await Cart.findByPk(req.params.id);
-    cart.destroy();
-    res.send(cart);
+    const cartToUpdate = await Cart.findByPk(req.body.id);
```

Comment:
I think we should find cart using userId:
`const cart = await Cart.findOne({
      where: {
        userId: req.user.id,
      },
    });`